packages/polka/src/index.ts

```diff
@@ -81,7 +83,10 @@ export default class Polka implements IModule, IServerHandler {
     const polkaConfig = config?.get<PolkaConfig>(this.name);
 
     const app = polka({
-      onError: (err, _req, res) => {
+      onError: (err, req, res) => {
+        const traceId = req.traceId || uuidv4();
+        err.traceId = traceId;
+
         // Regardless of what the error is we can notify newrelic of the error
         // Note: newrelic auto-instruments the http module, so any 4xx and 5xx will appear twice in your error dashboard
         nr?.api.noticeError(err);
```

Comment:
You should add customAttibute to the nr transaction during handle so that all errors automatically get traceId tagged.